added image links on README file

braulio94 opened the pull request:
Links for each image on `README.md` so people can access the source code easily.

braulio94 committed:
image links

braulio94 commented:
@googlebot `I signed it!`

mit-mit commented on README.md:
```diff
@@ -27,11 +27,21 @@ beautiful Material Design and
 Cupertino (iOS-flavor) widgets, rich motion APIs,
 smooth natural scrolling, and platform awareness.
 
+<a href="https://github.com/flutter/flutter/tree/master/examples/flutter_gallery/lib/demo/animation">
```
Can we use markdown formatting for these links, please?

braulio94 replied:
@mit-mit  by using markdown formatting, we'll have to sacrifice image sizing

braulio94 replied:
`[![alt](https://flutter.io/images/homepage/screenshot-2.png)](https://github.com/flutter/posse_gallery)`

meaning we can't add image size to markdown formatting

braulio94 replied:
@mit-mit  DONE SIR! 👍

mit-mit commented on README.md:
```diff
@@ -27,11 +27,21 @@ beautiful Material Design and
 <img src="https://flutter.io/images/homepage/screenshot-1.png" width="270" height="480" alt="Brand-first shopping design" align="left">
+</a>
+
+<a href="https://github.com/flutter/posse_gallery">
```
As above

mit-mit commented on README.md:
```diff
@@ -27,11 +27,21 @@ beautiful Material Design and
 <img src="https://flutter.io/images/homepage/screenshot-2.png" width="270" height="480" alt="Fitness app design">
+</a>
 
+<a href="https://github.com/flutter/flutter/blob/master/examples/flutter_gallery/lib/demo/contacts_demo.dart">
```
As above

mit-mit commented on README.md:
```diff
@@ -27,11 +27,21 @@ beautiful Material Design and
 <img src="https://flutter.io/images/homepage/screenshot-3.png" width="270" height="480" alt="Contact app design" align="left">
+</a>
+
+<a href="https://codelabs.developers.google.com/codelabs/flutter-firebase">
```
As above

mit-mit reviewed (changes requested):
Thanks for contributing! Some smaller comments

braulio94 committed:
image links with markdown formatting

changes of pull request [#13824](https://github.com/flutter/flutter/pull/13824)

braulio94 reviewed (commented):
image links with markdown formatting

mit-mit committed:
Remove empty lines

mit-mit commented:
Thanks for the contribution @braulio94 !

braulio94 commented:
Happy to help!
